src/Bridge/KdybyTranslation/TokenModifier/ParamsExtractorTokenModifier.php

```diff
@@ -44,21 +44,10 @@ private function createParamName(string $paramName): string
             return $this->staticParamsNameMap[$paramName];
         }
 
-        if (strpos($paramName, '(') !== false && strpos($paramName, ')') !== false) {
-            preg_match('/\((.*?)\)/', $paramName, $matches);
-            $paramName = $matches[1] ?? $paramName;
-        }
-
-        if (strpos($paramName, '$') !== 0) {
-            return $paramName;
-        }
-
-        $paramName = str_replace('$', '', $paramName);
-        if (strpos($paramName, '->') > 0 || strpos($paramName, '_') > 0) {
-            $paramName = str_replace(['->', '_'], '###DELIMITER###', $paramName);
-            $paramName = lcfirst(implode('', array_map('ucfirst', explode('###DELIMITER###', $paramName))));
-        }
-        return $paramName;
+        $paramName = str_replace(["$", "()", "->", "["], ["", "", "_", "_"], $paramName);
+        $paramName = preg_replace('/\W/', '', $paramName);
+        $result = preg_replace('/[A-Z]/', '_${0}', $paramName);
+        return strtolower(trim($result, '_'));
```

Comment:
nechceme mat tie parametre camelCase ci ako sa to vola?

Reply:
ja by som aj tie kluce robil underscored, nech to je rovnake s prekladmi.

Reply:
Pre mna to bola premenná tak som tak bral aj ten kľúč no :)